src/components/Articles.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils/api";
+
+function Articles() {
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        getArticles().then((articles) => {
+            setArticles(articles)
+        })
+    }, [])
+
+    return <section>
+        <ul>
+            {articles.map((article) => {
+                return <li key={article.article_id}>
+                    <img src={article.article_img_url}></img>
+                    <h2>{article.title}</h2>
+                </li>
```

Comment:
There's a lot of info from the objects that can be used! A bit picky but would love to see when the article was created and how you choose to display this. Will be handy for if you add a feature to sort articles by date 😄